pkgs/applications/audio/xmp/default.nix

```diff
@@ -2,11 +2,11 @@
 
 stdenv.mkDerivation rec {
   pname = "xmp";
-  version = "4.1.0";
+  version = "4.2.0";
 
   src = fetchurl {
```

Comment:
Would you please use `fetchFromGitHub`?

Reply:
The autogenerated tarball is missing a prebuilt `configure` script, while that's in the manually built tarball I specified here.

Reply:
How about using [autoreconfHook](https://nixos.org/manual/nixpkgs/stable/#setup-hook-autoconf)?
If not, leave a comment to avoid future confusion.

Reply:
Sure, that works. I thought maybe adding the extra dependency would be preferred, but I went ahead and switched to using that too. Pushed.